projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy-print.php

```diff
@@ -205,7 +205,8 @@ public function _print( $plain = false, $default_media = '' ) { // phpcs:ignore
 		}
 
 		if ( ! empty( $this->import ) ) {
-			for ( $i = 0, $size = count( $this->import ); $i < $size; $i++ ) {
+			$import_count = is_countable( $this->import ) ? count( $this->import ) : 0;
+			for ( $i = 0, $size = $import_count; $i < $size; $i++ ) {
```

Comment:
I don't think we need `$size` any more. I think it was only there before because we wanted to avoid calling `count( $this->import )` on every iteration of the loop.

```suggestion
			for ( $i = 0, $import_count; $i < $import_count; $i++ ) {
```

Reply:
Of course, thank you for catching it!